featurex/converters/api.py

```diff
@@ -29,6 +30,7 @@ def __init__(self, api_key=None):
         self.recognizer = sr.Recognizer()
         self.api_key = api_key
 
+    @abstractmethod
```

Comment:
I went with just making the properties abstract, seemed the most general way of doing this